### DISCUSSION TITLE
How to change sql mode in Planetscale?

### DISCUSSION THREAD

**Discussion by Rave2407** [planetscale/discussion#588] [Q&A] [answered]
I'have this error when i use GROUP BY in queryRawUnsafe
  
Expression #3 of SELECT list is not in GROUP BY clause and contains nonaggregated column \'test.v.imgUrl\' which is not functionally dependent on columns in GROUP BY clause; this is incompatible with sql_mode=only_full_group_by (errno 1055) (sqlstate 42000) 

with my research I saw that you had to declare

set sql_mode = REPLACE(@@sql_mode,'ONLY_FULL_GROUP_BY','');

but I don't understand where to declare it in my code.

**Comment by jgreet** [comment, marked as answer]
You'll have to change the SQL mode in every client session that's going to execute that query.

So if it's a common query you'll probably want to disable the `ONLY_FULL_GROUP_BY` mode in every connection, right after it's established. Alternatively, you could disable the `ONLY_FULL_GROUP_BY` mode right before executing the affected query.

See also https://github.com/planetscale/discussion/discussions/375
